admin/templates/configuration/etc/log4cxx.properties

```diff
@@ -5,6 +5,6 @@
 log4j.rootLogger=DEBUG, QLogger
 log4j.appender.QLogger=org.apache.log4j.ConsoleAppender
 log4j.appender.QLogger.layout=org.apache.log4j.PatternLayout
-log4j.appender.QLogger.layout.ConversionPattern=%d{MMdd HH:mm:ss.SSS} [%t] %-5p %c{2} (%F:%L) - %m%n
+log4j.appender.QLogger.layout.ConversionPattern=%d{MMdd HH:mm:ss.SSS} [PID:%X{PID}, %t] %-5p %c{2} (%F:%L) - %m%n
```

Comment:
I think it may be useful if we restart the service? Indeed all the log will be pushed to a centralized database (ElasticSearch) on the cluster, and it can be useful to have the pid here. Furthermore K.T. and Jacek agreed on this in DM-2430 (see ticket comments)